cross-project-tests/debuginfo-tests/dexter/dex/debugger/DAP.py

```diff
@@ -635,7 +635,7 @@ def launch(self, cmdline):
     def _post_step_hook(self):
         """Hook to be executed after completing a step request."""
 
-    def step(self):
+    def step_in(self):
```

Comment:
Why is this renaming necessary? We try to avoid changes to LLVM's code as much as possible.